src/custom/pages/Claim/index.tsx

```diff
@@ -23,7 +23,7 @@ import ClaimAddress from './ClaimAddress'
 import CanUserClaimMessage from './CanUserClaimMessage'
 import ClaimingStatus from './ClaimingStatus'
 import ClaimsTable from './ClaimsTable'
-import InvestmentFlow from './InvestmentFlow'
+import InvestmentFlow from './Investment'
```

Comment:
Why not calling the dir also `InvestmentFlow`?

Reply:
i did this, was tired, my bad